src/pages/settings/common/hooks/useHandleCompanySave.tsx

```diff
@@ -40,7 +40,10 @@ export function useHandleCompanySave() {
       return;
     }
 
-    !excludeToasters && toast.processing();
+    const adjustedExcludeToaster =
+      typeof excludeToasters === 'boolean' && !excludeToasters;
```

Comment:
@beganovich On the company settings page, we did not pass the `excludeToaster` value because we do not need it. Instead, by default, the `event` object is passed from the form's side. That is the reason for this expression.